.vscode/settings.json

```diff
@@ -4,5 +4,8 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": "always"
   },
-  "prettier.useEditorConfig": true
+  "prettier.useEditorConfig": true,
+  "[svelte]": {
+    "editor.defaultFormatter": "svelte.svelte-vscode"
```

Comment:
Måtte legge til dette for at .svelte-filene skulle autoformateres med prettier 🤔